pom.xml

```diff
@@ -28,9 +28,10 @@
     <version>40</version>
   </parent>
 
+  <!-- updated ztesler, 04-2016 -->
   <groupId>org.apereo.application</groupId>
   <artifactId>ImageQuiz</artifactId>
-  <version>1.0-SNAPSHOT</version>
+  <version>2.0</version>
```

Comment:
@bjagg should the version be `1.0.0` to fit the `major.minor.patch` format of [semver](http://semver.org/)?